cmd/tnf/fetch/fetch.go

```diff
@@ -15,22 +15,18 @@ import (
 )
 
 var (
-	filterCertifiedOperators = "&filter=organization==certified-operators"
-	containercatalogURL      = "https://catalog.redhat.com/api/containers/v1/images?page_size=%d&page=%d&filter=certified==true"
-	operatorcatalogURL       = "https://catalog.redhat.com/api/containers/v1/operators/bundles?"
-	helmcatalogURL           = "https://charts.openshift.io/index.yaml"
+	containersCatalogSizeURL = "https://catalog.redhat.com/api/containers/v1/images?filter=certified==true&page=0&include=total,page_size,data.docker_image_digest"
+	containersCatalogPageURL = "https://catalog.redhat.com/api/containers/v1/images?filter=certified==true&page_size=%d&page=%d&include=data.repositories,data.docker_image_digest,data.architecture"
+	operatorsCatalogSizeURL  = "https://catalog.redhat.com/api/containers/v1/operators/bundles?filter=organization==certified-operators"
+	operatorsCatalogPageURL  = "https://catalog.redhat.com/api/containers/v1/operators/bundles?filter=organization==certified-operators&page_size=%d&page=%d"
+	helmCatalogURL           = "https://charts.openshift.io/index.yaml"
```

Comment:
Why not also use include for the operator db:
    operatorsCatalogSizeURL  = "https://catalog.redhat.com/api/containers/v1/operators/bundles?filter=organization==certified-operators&include=total,page_size"
	operatorsCatalogPageURL  = "https://catalog.redhat.com/api/containers/v1/operators/bundles?filter=organization==certified-operators&page_size=%d&page=%d&include=data.csv_name,data.ocp_version,data.channel_name"